### PR TITLE
{tools}[foss/2023b] Optuna v3.6.1, redis-py v5.0.9

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3123.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/eed4db7cc126c7f0a9a1357469fd5f23 for a full test report.

File: easybuild/easyconfigs/r/redis-py/redis-py-5.0.9-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('typing-extensions', '4.10.0'),
+    ('Redis', '7.2.4'),
+]
+
+download_dep_fail = True
```
Comment (smoors) [inline easybuild/easyconfigs/r/redis-py/redis-py-5.0.9-GCCcore-13.2.0.eb:21]:
```suggestion
```

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Pull request merged**